src/views/mixed-content/government/index.js

```diff
@@ -7,11 +8,28 @@ import {Typography} from '@material-ui/core';
 const Government = () => {
   return (
     <Container data-testid="mixed-content-government">
-      <Typography variant="h4">Government</Typography>
-      <Typography variant="subtitle1">
-        Mixed content in governmental websites around the world
+      <Typography variant="h2" gutterBottom={true}>
+        Government
       </Typography>
-      <TopCountriesWithMoreWebsitesWithMixedContent />
+      <Typography variant="overline" gutterBottom={true}>
+        Summary of mixed content present in <b>government websites</b>.
+      </Typography>
+      <Typography variant="h6">Why analyze government websites?</Typography>
+      <Typography paragraph={true}>
+        Governmental websites are unfortunately the ones that contain the most
```

Comment:
Nit: I would probably suggest not making a conclusion of whether or not they are good attack targets, and instead suggest  something like "Government websites contain a large amount of mixed content, which can be particularly risky due to them also handling sensitive infromation."

Reply:
Thanks for letting me know! I'll convert this into an issue.